### DISCUSSION TITLE
Option to disable a specific earned badge

### DISCUSSION THREAD

**Discussion by ItsGosho** [community/community#18627] [Other Feature Feedback, Questions, & Ideas]
I'd like to suggest a option in the profile settings to disable/hide a specific badge from the ones that you have.

**Comment by benhoyt** [comment]
Part of the reason for this is the new "Pull Shark: x3" badge just seems a bit silly when you've been merging GitHub pull requests for 10 years... so I quite like my "Mars Helicopter" achievement, but definitely don't want the Pull Shark one.

Edit: Oh, I see this is already being discussed at https://github.com/github-community/community/discussions/18345

**Comment by cyai** [comment]
You can remove achievements entirely from your profile at https://github.com/settings/profile

![image](https://user-images.githubusercontent.com/83634399/174118579-f2550b05-5477-42d7-a065-f5aa3f21dfad.png)

> NOTE: Turning off specific achievements would be a nice feat, but for now you can only turn off everything.
